recipes-kernel/linux/files/bcm2712-raspberrypi5-xen.dts

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
Why is this file needed? Now we add all Xen cfg with boot scripts.

Reply:
I don't think it is good idea. This noticeable reduce readability.
I prefer modify some items in boot script.